packages/web/src/pages/invalidEmail.tsx

```diff
@@ -0,0 +1,30 @@
+import { NextPage } from 'next';
+import { Button, Heading, HStack, Text, VStack } from '@chakra-ui/react';
+import { MarketingLayout } from '../components/Layout';
+
+const InvalidEmail: NextPage = () => {
+  return (
+    <MarketingLayout>
+      <VStack alignItems="center" paddingTop={8} w="100%" maxW="1200px" marginX="auto">
```

Comment:
You should be using `align="stretch"` here to accommodate the horizontal width stuff; the way it's setup now might cause problems on different screen sizes